### DISCUSSION TITLE
Is it possible to download whole folders from bucket?

### DISCUSSION THREAD

**Discussion by YO-SC** [supabase/supabase#26893] [Questions]
The title says it all. I have read the [documentation](https://supabase.com/docs/reference/javascript) but I can't find anything like that, hence I am here asking. Most probably is not a feature, but I ask just in case 😇.

To elaborate, I am interested in downloading the folder itself, instead of the files contained within.

**Comment by GaryAustin1** [comment]
Not sure what a folder is to you.  There is no such thing as folder in reality, it is just a concept using string processing on the names between /'s.

The .list() command has an option to get the files in different levels of "folders".
![image](https://github.com/supabase/supabase/assets/54564956/65bf3237-abbe-41e4-9102-4be5fa626ae0)
